chainer/training/updater.py

```diff
@@ -289,8 +289,8 @@ def update_core(self):
             in_arrays_list[key] = self.converter(
                 batch[i::n], self._devices[key])
 
-        for model in six.itervalues(self._models):
-            model.cleargrads()
+        # for model in six.itervalues(self._models):
```

Comment:
I `--amend`-ed the previous commit to exclude updater.py, as it was not my intention to include it in the PR.